PIL/ImageShow.py

```diff
@@ -69,8 +69,9 @@ def show(self, image, **options):
             # FIXME: auto-contrast if max() > 255?
         else:
             base = Image.getmodebase(image.mode)
-        if base != image.mode and image.mode != "1":
+        if base != image.mode and image.mode != "1" and image.mode != "RGBA":
             image = image.convert(base)
```

Comment:
IMHO, as long as the windows viewer does not choke on the alpha-enabled BMP, it is no worse than the current situation, which is already wrong.